src/app/pages/vm/vm-cards/vm-cards.component.ts

```diff
@@ -523,12 +524,14 @@ export class VmCardsComponent implements OnInit,OnDestroy {
     }
      else {
       if(poweroff){
-         this.off_text =  `Do you wish to Power Off ${vm.name} ?`;
+         this.off_text =  `Power off to ${vm.name} ?`;
+         this.off_button_text = 'Power off';
       }
       else {
-        this.off_text =  `Do you wish to Turn Off ${vm.name} ?`;
+        this.off_text =  `Stop virtual machine ${vm.name} ?`;
```

Comment:
As above, I pasted the actual code to be copied.

Reply:
what delimiters do you want ?